frontend/sozisel-app/src/components/TemplatesList/TemplateCard/TemplateCard.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../../styles/styles";
+
+.templateCard{
+    display: flex;
+    flex-direction: row;
+    padding: $spacing-05;
+    margin-bottom: $spacing-04;
+    background: "#f0f0f0";
```

Comment:
I think that we will probably want to keep those colors as variables same as with spacings. Easier to reuse.